pokepro/src/components/Card/card.css

```diff
@@ -0,0 +1,28 @@
+.individual_card_container{
+    width: 15em;
+    height: 20em;
+    border: 2px solid gold;
+    display: flex;
+    flex-direction: column;
+    align-items:center;
+    /* justify-content:center; */
+    padding:1px;
+    margin: 1rem;
+    background-color: rgb(254, 254, 252);
+
+}
+.individual_card-image-container{
+    /* background-color: aqua; */
+    width:90%;
+    display:flex;
+    /* align-items: center; */
+    justify-content: center;
+    height: 40%;
+    margin-top: 20px;}
+.pokeImage{
+    width:100%;
+    height:100%;
+}
```

Comment:
I don't think we would require this div and this class here, we can directly use the image tag and manipulate the style on the image tag.